BrightFuturesTests/ResultTests.swift

```diff
@@ -21,8 +21,18 @@
 // SOFTWARE.
 
 import XCTest
+import Result
 import BrightFutures
 
+extension Result {
+    var isSuccess: Bool {
```

Comment:
I am not using `isSuccess` and `isFailure` in my code either, so maybe we can just remove them. What do you think?